go/vt/vtgate/planbuilder/operators/vindex.go

```diff
@@ -49,7 +48,7 @@ type (
 	}
 )
 
-const vindexUnsupported = "unsupported: where clause for vindex function must be of the form id = <val> or id in(<val>,...)"
+const vindexUnsupported = "where clause for vindex function must be of the form id = <val> or id in(<val>,...)"
```

Comment:
```suggestion
const vindexUnsupported = "WHERE clause for vindex function must be of the form id = <val> or id in(<val>,...)"
```

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)